### DISCUSSION TITLE
Custom CSS for only 1 view mode

### DISCUSSION THREAD

**Discussion by dsferruzza** [Athou/commafeed#1232] [Q&A] [answered]
Hi!

As far as I understand, there are 4 view modes: https://github.com/Athou/commafeed/blob/master/commafeed-client/src/app/types.ts#L291
I would like to add some custom CSS for only one of them, but I could not find a way to apply my custom CSS only when this view mode is active.
Is there a class or something that is applied to the feed entries depending on the current view mode?

Have a nice day!

**Comment by Athou** [comment, marked as answer]
There's one now https://github.com/Athou/commafeed/commit/b0890df2f378e49d5e76d424ac14da2d9fe9a58b :)

**Reply by Athou** [reply]
I just released 4.3.0 that includes this.

**Reply by dsferruzza** [reply]
Thank you!

**Reply by Athou** [reply]
Just a heads up, a lot of CSS classes were added throughout the code [here](https://github.com/Athou/commafeed/commit/d417655a865026e5967da80525f81ba4c38ad027) and [here](https://github.com/Athou/commafeed/commit/b5c197f49928c046daf7a976a961fe312353234a) to ease customization. In order to help differentiate CSS classes that can be used for customization from CSS classes that are auto-generated, they have been prefixed with `cf-`. Therefore, I renamed `view-mode-*` to `cf-view-mode-*` for consistency. This will be part of the next version of CommaFeed.
